scm/src/GFS_typedefs.meta

```diff
@@ -607,6 +614,14 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[sncovr_ice]
+  standard_name = surface_snow_area_fraction_over_ice
+  long_name = surface snow area fraction over ice
+  units = frac
+  dimensions = (horizontal_dimension)
```

Comment:
Why is this `horizontal_dimension`? Should be `horizontal_loop_extent` I think. Same for a few other variables in this `.meta` file (search & replace?).

Reply:
Yep, thanks for catching this mistake. I'll push a commit once successfully recompiled.

Reply:
Done.